assets/logo/README.md

```diff
@@ -0,0 +1,14 @@
+# Our logo assets
+These logos/icons are free for use and are released under the [LICENSE](../../LICENSE) of this repository!  
```

Comment:
```suggestion
These logos/icons are free for use and are released under the [LICENSE](../../LICENSE) of this repository.
```

Reply:
I actually wanted the `!` to underline our free to use intention.
Also the newline is just prettier...